rotkehlchen/chain/ethereum/modules/balancer/balancer.py

```diff
@@ -815,33 +802,6 @@ def _get_protocol_balance_graph(
         )
         return protocol_balance
 
-    def _get_token_price_at_timestamp_zero_if_error(
-            self,
-            token: EvmToken,
-            timestamp: Timestamp,
-    ) -> Price:
-        if token.has_oracle():
-            usd_price = query_usd_price_or_use_default(
-                asset=token,
-                time=timestamp,
-                default_value=ZERO,
-                location=str(Location.BALANCER),
-            )
-        else:
```

Comment:
what about removing this suppression part? Why is it okay?

Reply:
in that case it will use the defillama oracle or otherwise will default to 0. To query historical prices we would need archive logic to do it
the supress logic was there to go and find a price if coingecko/cryptocompare didn't have any